documentation/library_guide/macros.rst

```diff
@@ -64,11 +64,34 @@ Macro                              Description
 ---------------------------------- ------------------------------
 ``ONEDPL_USE_TBB_BACKEND``         This macro controls the use of |onetbb_long| or
                                    |tbb_long| for parallel policies.
-                                   When the macro is set to 0, algorithms with the ``par`` and ``par_unseq`` policies are only
-                                   executed by the calling thread. This is recommended for code that should not depend on the
-                                   presence of the |onetbb_short| or |tbb_short| library. When the macro is not defined (by default)
-                                   or evaluates to a non-zero value,
+                                   When the macro is set to 0, algorithms with the ``par`` and ``par_unseq`` policies do not
```

Comment:
I would explain that par and par_unseq are collectively called "parallel policies" just once at the line 66, and avoid repeatedly mentioning them later.

Reply:
Done. Intentionally left the mentioning of ``par`` and ``par_unseq`` for both ``TBB`` and ``OPENMP`` macros because they could be read let's say starting from ``OPEN_MP`` one